### ISSUE TITLE
Buy Network Firewall for ESI core network

### ISSUE THREAD

**Issue by hakasapl** [CCI-MOC/ops-issues#995]
Buy 1x of these netgate firewalls: https://shop.netgate.com/products/1541-max-pfsense?variant=32156884369523

**Comment by joachimweyl** [comment]
Blocked on having a meeting with Scott

**Comment by taramoran** [comment]
@joachimweyl to confirm, I shouldn't order until you all meet w Scott?

**Comment by msdisme** [comment]
@hakasapl MAX or RAID model? plan to use pfsense or TAC?

**Comment by hakasapl** [comment]
Didn't realize there was a raid model. Let's do these parameters:

* RAID
* PfSense+
* Expansion Card: 4-Port 10GbE Fiber SFP+ NIC

This link should get you to that config: https://shop.netgate.com/products/1541-raid-pfsense?variant=39280942743667

**Comment by msdisme** [comment]
OK - need to figure out timing based on when we have enough credits with Flax vs. straight purchase - will discuss with @neclinton  monday morning

**Comment by msdisme** [comment]
Order placed - moving to in review till it gets here : Netgate
ORDER SO23-95568

**Comment by naved001** [comment]
@hakasapl maybe I am little to ask this, but did we only order a single firewall? I have been reading up their documentation, and apparently we can run a pair of these for high availability. Now, I don't see that specific model available to purchase, and their documentations say that for HA configuration the hardware must match (though I am seeing if there are workarounds).

**Comment by hakasapl** [comment]
@naved001 We only bought one but adding another will always be possible. The software itself doesn't even need to support HA if all we're looking for is redundancy and not more bandwidth (which I think is the case for public networks). In that case we would have both firewalls peer with a provider and setup VRRP on either the firewalls or the adjacent switch, which will give us the redundancy.

Since the software supports HA too we can likely do more fancy things like HA vpn and such, but the most important requirement of HA public networks can be fulfilled without the firewalls necessarily supporting it.

**Comment by naved001** [comment]
@hakasapl okay, ultimately I'd like to use the firewalls for NAT for some internal networks, and run a VPN to access various IPMI networks as well.

**Comment by hakasapl** [comment]
@naved001 yeah makes sense. I guess we're blocked on BU network for that still but I will circle back
